libnmstate/nm/route.py

```diff
@@ -16,12 +16,18 @@
 #
 
 from operator import itemgetter
-from libnmstate.error import NmstateInternalError
-from libnmstate.schema import Route
+import socket
 
 from libnmstate import iplib
+from libnmstate.error import NmstateInternalError
```

Comment:
nit: Can't you import error and use it with a context?
`error.NmstateInternalError`

Reply:
Any particular reason for this?

Reply:
Why is it not nice? Seems to be more like a matter of taste.

Reply:
That is why it is not a must :)
In many cases, having the module as the context/namespace and avoiding for each additional error usage to add another import line is nicer IMO.

Reply:
After thinking some more I believe that using `from libnmstate import error` pollutes the global namespace with a generic word `error` and since it is easy to skip over many import lines I actually prefer the individual imports. Maybe moving the exceptions to libnmstate directly would make it nice enough again but I am not sure.